cmd/podman/network_create.go

```diff
@@ -46,7 +46,7 @@ func init() {
 	// TODO enable when IPv6 is working
 	//flags.BoolVar(&networkCreateCommand.IPV6, "IPv6", false, "enable IPv6 networking")
 	flags.IPNetVar(&networkCreateCommand.Network, "subnet", net.IPNet{}, "subnet in CIDR format")
-
+	flags.BoolVar(&networkCreateCommand.DisableDNS, "disabledns", false, "disable dns plugin")
```

Comment:
To match format with other multi-word flags, this should probably be `--disable-dns`